### ISSUE TITLE
Deap 1.3.1 lead to error

### ISSUE THREAD

**Issue by sqbl** [BoostV/process-optimizer-api#51]
Hi,
Installing the docker image on raspberry 4 leads to an error: `use_2to3 is invalid`.

Internet suggest to downgrade to setuptools==75.5.0, this didn't help.
DEAP package seems aware of issue and version 1.3.3 has it solved. By manually changing requirements-freeze.txt to deap==1.3.3 seems to remove issue.
I guess it would make most sense to change the dependence at "processoptimizer" level. So in other words, this issue is just for info. Can be closed when the frozen requirements have been changed

**Comment by sqbl** [comment]
https://github.com/DEAP/deap/commit/7d372b17b5359a33f46c444b313b12f8e74f8e27

**Comment by sqbl** [comment]
https://stackoverflow.com/questions/69100275/error-while-downloading-the-requirements-using-pip-install-setup-command-use-2

**Comment by sqbl** [comment]
Deap is pinned to 1.3.3 in ProcessOptimizer, now. Can we up the pinned version in this repo?

**Comment by langdal** [comment]
Is the pinned version published to pip?
In PR #50 ProcessOptimizer has been set to ProcessOptimizer[browniebee]==0.7.6 however the tests fail because of missing module `ProcessOptimizer.model_systems`

```
============================= test session starts ==============================
[8](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:9)
platform linux -- Python 3.9.15, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
[9](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:10)
rootdir: /home/runner/work/process-optimizer-api/process-optimizer-api, configfile: pytest.ini
[10](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:11)
collected 2 items / 1 error / 1 selected
[11](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:12)

[12](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:13)
==================================== ERRORS ====================================
[13](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:14)
___________________ ERROR collecting tests/test_optimizer.py ___________________
[14](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:15)
ImportError while importing test module '/home/runner/work/process-optimizer-api/process-optimizer-api/tests/test_optimizer.py'.
[15](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:16)
Hint: make sure your test modules/packages have valid Python names.
[16](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:17)
Traceback:
[17](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:18)
/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/importlib/__init__.py:127: in import_module
[18](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:19)
    return _bootstrap._gcd_import(name[level:], package, level)
[19](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:20)
tests/test_optimizer.py:4: in <module>
[20](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:21)
    from optimizerapi import optimizer
[21](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:22)
optimizerapi/optimizer.py:19: in <module>
[22](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:23)
    from ProcessOptimizer import Optimizer, expected_minimum
[23](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:24)
/opt/hostedtoolcache/Python/3.9.15/x64/lib/python3.9/site-packages/ProcessOptimizer/__init__.py:18: in <module>
[24](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:25)
    from .model_systems import benchmarks
[25](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:26)
E   ModuleNotFoundError: No module named 'ProcessOptimizer.model_systems'
[26](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:27)
=========================== short test summary info ============================
[27](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:28)
ERROR tests/test_optimizer.py
[28](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:29)
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
[29](https://github.com/BoostV/process-optimizer-api/actions/runs/3393885013/jobs/5990885344#step:6:30)
=============================== 1 error in 0.68s ===============================
```

**Comment by sqbl** [comment]
Hi,
It is now pinned to 1.3.3. We had a small mishap in 0.7.6 regarding the model systems, but that should have been fixed by 0.7.7, which is pip'able.

Best,
Søren
